wallet/wallet-controller/src/lib.rs

```diff
@@ -125,7 +129,16 @@ impl<T: NodeInterface + Clone + Send + Sync + 'static> Controller<T> {
 
     pub fn get_balance(&self) -> Result<(Amount, BTreeMap<TokenId, Amount>), ControllerError<T>> {
         self.wallet
-            .get_balance(DEFAULT_ACCOUNT_INDEX)
+            .get_balance(
+                DEFAULT_ACCOUNT_INDEX,
+                UtxoType::Transfer | UtxoType::LockThenTransfer,
+            )
+            .map_err(ControllerError::WalletError)
+    }
+
+    pub fn get_utxos(&self) -> Result<BTreeMap<OutPoint, TxOutput>, ControllerError<T>> {
```

Comment:
get_all_utxos(), since we do have a filter?

Even better, keep the name, and pass UtxosTypes as parameter. There can be Default impl for UtxosTypes that's equivalent to UtxosTypes::ALL.